# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/heap-analysis/findPath.js
# Commit: 11fc8f557a2a
# Full Hash: 11fc8f557a2a7cf3c89e7c1396316aabd9725778
# Author: Jan de Mooij <jdemooij@mozilla.com>
# Date: 2021-03-06 09:33:16
# Regressor Bug: 1689413
# File Overlap Count: 1
# Description:
#   Bug 1689413 part 6 - Move Realm* from ObjectGroup to BaseShape. r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D106976
# ==============================================================================

diff -r 9bfa6023a553 -r 11fc8f557a2a js/src/jit-test/tests/heap-analysis/findPath.js
--- a/js/src/jit-test/tests/heap-analysis/findPath.js	Sat Mar 06 01:05:20 2021 +0000
+++ b/js/src/jit-test/tests/heap-analysis/findPath.js	Sat Mar 06 01:05:21 2021 +0000
@@ -35,8 +35,9 @@
   .assert(findPath(gc, o));
 print(JSON.stringify(findPath(gc, o)));
 
-Match.Pattern([{node: {}, edge: "group"},
-               {node: Match.Pattern.ANY, edge: "group_global"},
+Match.Pattern([{node: {}, edge: "shape"},
+               {node: Match.Pattern.ANY, edge: "base"},
+               {node: Match.Pattern.ANY, edge: "baseshape_global"},
                {node: {}, edge: "o"}])
   .assert(findPath(o, o));
 print(findPath(o, o).map((e) => e.edge).toString());